packages/storybook/src/stories/grid-rows.stories.tsx

```diff
@@ -176,9 +176,19 @@ const columns: any[] = [
 ];
 
 export function ScrollIssue() {
+  const apiRef = useGridApiRef();
+
+  React.useEffect(() => {
+    const timeout = setTimeout(() => {
```

Comment:
The screenshot is waiting in a useEffect: https://github.com/mui-org/material-ui-x/pull/1081/files#diff-df56773d39f0fa9c8f39611f1993bf7df81d046c5d055a47365fccb494407de0R63. It's not waiting for this timeout to run. It won't screenshot it correctly;

Reply:
Are we sure it should be a screenshot to best test it? Would assorting the dimensions work better?

Reply:
we have to wait until everything is rendered before we can move the scroll down. It seems to only work in a timeout.
I tried with useLayoutEffect...